hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml

```diff
@@ -18,4 +18,85 @@
 
 -->
 <server xmlns="urn:jboss:domain:4.0">
+
+  <extensions>
+        <?EXTENSIONS?>
+  </extensions>
+
+  <management>
```

Comment:
@bobmcwhirter thanks for the info.

@jmazzitelli , as I said, I vote for merging this as is now and experiment with wf-core later, when we have split the message classes to a new JMS independent maven module.